### PR TITLE
Feature: Convert DriveHandler to Files.App helpers

### PR DESCRIPTION
**Pull request by lukeblevins**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Helps with #9858 

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**

### CODE REVIEW

File: src/Files.App/UserControls/SidebarControl.xaml.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@
 using CommunityToolkit.WinUI.UI;
 using Microsoft.UI.Input;
 using UWPToWinAppSDKUpgradeHelpers;
+using Vanara.PInvoke;
```
Comment (yaichenbaum) [inline src/Files.App/UserControls/SidebarControl.xaml.cs:32]: Is there supposed to be an extra space?

**Comment by yaichenbaum** [comment]
@hez2010 can you look this over (no need to compile here)?

File: src/Files.App/ServicesImplementation/VolumeInfoFactory.cs [hunk 2]
```diff
@@ -18,25 +15,7 @@ public async Task<VolumeInfo> BuildVolumeInfo(string driveName)
 
         private async Task<string> GetVolumeID(string driveName)
```
Comment (hez2010) [inline src/Files.App/ServicesImplementation/VolumeInfoFactory.cs:16]: The `async Task<string>` here is redundant. Can we change it to a sync method?

**Review by hez2010 — DISMISSED** [review]
Otherwise LGTM.

**Comment by d2dyno1** [comment]
Could you remove conflicts? 🙂

File: src/Files.App/ServicesImplementation/VolumeInfoFactory.cs (hunk 2 above)
Comment (lukeblevins) [reply src/Files.App/ServicesImplementation/VolumeInfoFactory.cs]: > Can we change it to a non-async method which simply returns a string?

Done!

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**